### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by maxsupera**
fixed codec slug in config. added github actions to ignore.

### CODE REVIEW

**Comment by joepetrowski** [comment]
Why are you adding `.github` to the gitignore? We do want to track the template in that folder.

**Comment by maxsupera** [comment]
note: those additional 2 commits fix the markdown images loading for the frontend

**Comment by thisisvilla** [comment]
> Why are you adding `.github` to the gitignore? We do want to track the template in that folder.

the github actions (which rebuild the frontend) can be edited directly in github and left in the master branch. there is no reason users should be seeing or doing pull requests on any of those files, and we definitely do not want them to.

File: current/runtime/frame.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Some pallets will be sufficiently general-purpose to be reused in many blockchai
 The Assets pallet is a simple, secure module for dealing with fungible assets.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_assets/index.html)
-* [Source](https://github.com/paritytech/substrate/blob/master/frame/assets/src/lib.rs)
+* [Source](https://github.com/paritytech/substrate/blob/master/frame../../assets/src/lib.rs)
```
Comment (joepetrowski) [inline current/runtime/frame.md]: This doesn't seem right

Comment (maxsupera) [reply current/runtime/frame.md]: ah shit find and replace failed me

Comment (maxsupera) [reply current/runtime/frame.md]: thats been fixed

**Comment by maxsupera** [comment]
ok i removed the .gitignore. i think this one is good to go

**Review by joepetrowski — APPROVED** [review]

**Pull request merged**